image_analyzer/core/chunking/chunk_processor.py

```
@@ -7,17 +7,18 @@
 from PIL import Image
 
 from ..image_analyzer import analyze_image_with_ollama
-from ..text_summarizer import summarize_text
+from ..text_summarizer import summarize_text, assemble_text_chunks
 from .image_chunker import chunk_image, save_image_chunks
```

Ruff (0.12.2)

`.image_chunker.save_image_chunks` imported but unused

Remove unused import: `.image_chunker.save_image_chunks`

(F401)